tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php

```diff
@@ -117,6 +117,24 @@ public static function clearDownloads(): void
         }
     }
 
+    /**
+     * @AfterFeature @clear-img-after-feature
+     */
+    public static function clearImgDir(): void
+    {
+        //@todo: its not working because img/* directories permissions. Those are created with Image->createImgFolder()
```

Comment:
Can you elaborate 🤔 ?
It makes me think `Image->createImgFolder()` is buggy

Reply:
false alarm. My local environment permissions fault.